backend/oasst_backend/models/message_tree_state.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+from typing import Optional
+from uuid import UUID, uuid4
+
+import sqlalchemy as sa
+import sqlalchemy.dialects.postgresql as pg
+from sqlalchemy import false
+from sqlmodel import Field, Index, SQLModel
+
+INITIAL = "initial"
+BREEDING_PHASE = "breeding_phase"
```

Comment:
these should probably be put into a str-enum (see other cases where enums are used in the prj)

Reply:
okay, Ill make the changes

Reply:
ok rewrote the Enum class for states and updated VALID_STATES